src/components/Plants/usePlantPics.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const API_URL =
+  'https://www.flickr.com/services/rest/?method=flickr.photos.search&format=json&nojsoncallback=1';
+const API_KEY = process.env.NEXT_PUBLIC_FLICKR_API_KEY;
+
+// TODO add return type
+export default function usePlantPics(name: string) {
+  let [photos, setPhotos] = useState([]);
+
+  // TODO: use fetch instead?
+  useEffect(() => {
+    axios
+      .get(
+        `${API_URL}&api_key=${API_KEY}&sort=relevance&text=${name}&license=1,2,3,4,5,6,9,10&per_page=12`,
```

Comment:
not fetching pics with All rights reserved, just creative commons and public domain (will look into them closely to decide which ones leave here)

Reply:
fikus benjamina for sure needs a different query text, rest seems ok 😂
<img width="856" alt="Screenshot 2020-11-29 at 12 29 31" src="https://user-images.githubusercontent.com/9083292/100540601-8a06e180-323e-11eb-9eda-bf0e93d65f36.png">

Reply:
https://www.flickr.com/services/api/flickr.photos.licenses.getInfo.html

Reply:
https://www.flickr.com/services/api/flickr.photos.search.html

Reply:
(it's not outdated, just change something in this line and github thinks these comments are not relevant anymore)

Reply:
I added tags, plus a `query` field to use instead of Latin name if the results are... hm... not plant related ;)